src/docs/asciidoc/community/documentation.adoc

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+:imagesdir: ../images
+
+= Documentation Guidelines
+
+The aim of this section is to explain how the documentation is currently organized and to give a few guidelines on how
+it should be written.
+
+== Structure
+
+All the sources for the AsciiDoc documentation published on our link:https://opfab.github.io/[website] are found under
+the link:https://github.com/opfab/operatorfabric-core/tree/master/src/docs/asciidoc[src/docs/asciidoc] folder in the
+operatorfabric-core repository.
+
+Its is organized into several folders (architecture documentation, deployment documentation, etc.). Each of these
```

Comment:
'It's organized'' || 'It is organized' ?

Reply:
Good eye for typos ;-) ! Thanks